src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,40 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Superclass for the Phone,Email and Address classes
```

Comment:
It's not necessary to mention the child classes. By doing so, you need to keep updating the header comment for this class.